index.html

```diff
@@ -325,11 +300,10 @@ <h4>Introduction</h4>
           * `n! = n * (n-1) * (n-2) * ... * 1`
           * `n! = n * (n-1)!`
 
-          ```clj
-          (defn factorial [n]
-            (if (<= n 1)
-              1
-              (*' n (factorial (- n 1)))))
+          ```haskell
+          factorial n =
+              if n <= 1 then 1
```

Comment:
maybe a new line before `then`?

Reply:
I don't think it's necessary. An ideal form for Haskell is an equation, like:

```haskell
factorial 0 = 1
factorial n = n * factorial (n - 1)
```
But there is no slide for pattern matching yet, and the current form is closest to that.